packages/app/src/setupTests.ts

```diff
@@ -53,3 +55,27 @@ afterEach(() => {
     delete globalThis.jest;
   }
 });
+
+// Mock the size of elements measured with `useMeasure`
+// Use attribute `data-test-size="<width>,<height>"` to specify the element's size
+vi.mock('@react-hookz/web', async (importOriginal) => {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-imports
+  const mod = await importOriginal<typeof import('@react-hookz/web')>();
+
+  return {
+    ...mod,
+    useMeasure: () => {
```

Comment:
`useMeasure` relies on `ResizeObserver` which is not supported by JSDOM. As a result, the `MatrixVis` visualization used to render an empty container. I now fake the width and height of the container so that `react-window` renders something.